src/Features/Core/Portable/CodeRefactorings/AbstractRefactoringHelpersService.cs

```diff
@@ -336,13 +343,26 @@ protected virtual SyntaxNode DefaultNodeExtractor(SyntaxNode node, ISyntaxFactsS
                 return rightSide;
             }
 
+            // VB's arguments can have identifiers nested in ModifiedArgument -> we want
+            // indentifiers to represent parent node -> need to extract.
+            if (syntaxFacts.IsParametersIdentifier(node))
```

Comment:
odd name.  why not `IsParameterIdentifier`?  Note: naming for ISyntaxFacts is `IsIdentifierOfParameter`.  This naming better helpes with things like `IsStatementExpressoinExpression`.  i.e. `IsExpressionOfStatementExpression` is much clearer than the former.